tests/ui/conftest.py

```diff
@@ -1,11 +1,24 @@
 import datetime
+import json
 import os
 import urlparse
 
-from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
 import jwt
 import pytest
 import requests
+from django.core.management import call_command
+from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
+from olympia.amo.tests import create_switch
+from pytest_django import live_server_helper
+
+
+@pytest.fixture
+def local_base_url(live_server):
```

Comment:
Ideally we don't want to start a live server unless needed. We should use the value of the session scoped `base_url` fixture to determine if we should start a live server. We shouldn't assume that `localhost` means start a live server, as it's possible the user wants to run tests against a server that's been independently started locally. Unless there's a reason not to, I would infer that no base URL means start a live server, so this fixture would look something like:

```python
@pytest.fixture(scope='function')
def my_base_url(base_url, request):
    return base_url or request.getfixturevalue("live_server").url
```